### ISSUE TITLE
lfrc sources shortcutrc

### ISSUE THREAD

**Issue by hnsstrk** [LukeSmithxyz/voidrice#1281]
In the last to lines of the lfrc shortcutrc is sourced but the file doesn't exist. 

```
# Source Bookmarks
source "~/.config/lf/shortcutrc"
```

**Comment by rxmmah** [comment]
its still an problem :(

**Comment by corbin-ch** [comment]
> its still a problem :(

i'm not an expert on these dots but i took a quick peek around.

`~/.config/lf/shortcutrc` is created by the `~/.local/bin/shortcuts` script. this script is automatically run from `~/.config/shell/profile` if `~/.config/shell/shortcutrc` does not exist -- and it doesn't look like `~/.config/shell/shortcutrc` exists until the `shortcuts` script is run, at which point both `shortcutrc` files would be created

tldr: unless i'm reading this wrong, it does not appear to be a problem

hope this helps!
